stampit.js

```diff
@@ -189,16 +191,21 @@ isStamp = function isStamp(obj) {
  * @return {Function}             A composable stampit factory
  *                                (aka stamp).
  */
-convertConstructor = function convertConstructor(Constructor) {
-  return stampit().methods(Constructor.prototype).enclose(Constructor);
-};
+function convertConstructor(Constructor) {
+  var stamp = stampit();
+  mixer.mixInChainFunctions(stamp.fixed.methods, Constructor.prototype);
+  mixer.mixIn(stamp.fixed.state, Constructor);
```

Comment:
What's the point of this? Very few Constructor functions have properties attached to them, and even if they did, it would be more appropriate to copy them to the stamp function itself than it would be to copy it as instance state.